executor/executor.go

```diff
@@ -1,24 +1,28 @@
 package executor
 
 import (
+	"bufio"
 	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
 	"os/exec"
-	"syscall"
+	"regexp"
+	"strconv"
+	"strings"
 
+	"github.com/kr/pty"
```

Comment:
Hey @jer, couldn't get it to work. Having same issues as these people here:
https://github.com/Masterminds/glide/issues/298
https://github.com/Masterminds/glide/issues/577

Those issues are still opened. Also, looks like kr/pty is not being developed. Only 1 release so far. Maybe we can make this work in the next PR?